packages/io/svg-serializer/index.js

```diff
@@ -30,40 +27,32 @@ const mimeType = 'image/svg+xml'
  * @param {Object|Array} objects - objects to serialize as SVG
  * @returns {Array} serialized contents, SVG format
  */
-const serialize = (...params) => {
-  let options = {}
-  let objects
-  if (params.length === 0) {
-    throw new Error('no arguments supplied to serialize function !')
-  } else if (params.length === 1) {
-    // assumed to be object(s)
-    objects = Array.isArray(params[0]) ? params[0] : params
-  } else if (params.length > 1) {
-    options = params[0]
-    objects = params[1]
-  }
-  // make sure we always deal with arrays of objects as inputs
-  objects = toArray(objects)
-  console.log('params', params)
-  console.log('options', options)
-  console.log('objects', objects)
+const serialize = (options, ...objects) => {
   const defaults = {
-    statusCallback: null,
     unit: 'mm', // em | ex | px | in | cm | mm | pt | pc
-    decimals: 10000
+    decimals: 10000,
+    statusCallback: null
   }
   options = Object.assign({}, defaults, options)
 
+  objects = utils.flatten(objects)
```

Comment:
this will not work as expected: 
- toArray() ensures we always have arrays
- flatten() flattens multi dimensional arrays in one dimensional ones

Reply:
Should work...

`const serialize = (options, ...objects) => {`

Reply:
ahh yes, that is a VERY good point ! my bad ! Had to double check with a test function but rest parameters indeed make sure of that !